go/vt/tabletserver/query_executor.go

```diff
@@ -49,6 +49,18 @@ func (qre *QueryExecutor) Execute() (reply *mproto.QueryResult, err error) {
 	defer func(start time.Time) {
 		duration := time.Now().Sub(start)
 		qre.qe.queryServiceStats.QueryStats.Add(planName, duration)
+
+		username := "unknown"
+		if callerID := callerid.ImmediateCallerIDFromContext(qre.ctx); callerID != nil {
```

Comment:
Please use the APIs in callerid.go, which eliminate the need to judge if callerid == nil